content/references/markdown/index.md

```diff
@@ -4,12 +4,164 @@ topic: markdown
 next: /references/markdown/extensions/
 ---
 
+
+
 # Basic Markdown Syntax
 
 Codewars supports [Markdown][wiki-markdown].
 
 More specifically, [GitHub Flavored Markdown][gfm] (strict superset of [CommonMark][common-mark]) and few [Codewars extensions][extensions].
 
+You may find below a summary of the most used ones.
+
+
+
+
+## Emphasis
+
+```markdown
+_This is italic_
+*This is also italic*
+
+__This is bold__
+**This is bold too**
+
+___here, italic and bold___
+***here, italic and bold***
+```
+Note that `*You **can** combine them*`: *You **can** combine them*.
+
+
+
+
+## Headers
+
+```markdown
+# This is &lt;h1&gt;
+## This is &lt;h2&gt;
+###### This is &lt;h6&gt;
+```
+
+
+
+## Code blocks
+
+To get text displayed without the renderer playing with the indentation or whitespaces, code blocks
+
+### Inline code blocks:
+
+To highlight code inside a sentence, use simple backticks  ``` `print("This is some code")` ```.
+
+### Mutliline code blocks:
+
+For multiline code blocks, use triple backticks.
+
+Constraints:
+   - Put an empty line before and after the code block itself, so that it's rendered correctly
```

Comment:
Is this Codewars' bug? I don't think empty lines are necessary.

Reply:
The only thing I know is that I'm almost always lost about how to do this, between gitter, CW and here... :/

But currently, on codewars, it's definitely required otherwise youget some mess like that: https://www.codewars.com/kata/56941f177c0a52aef50000a2

Reply:
That's because of all the mess it has:

![image](https://user-images.githubusercontent.com/639336/88430456-522df000-cdad-11ea-9f59-a16e8ecd700b.png)

Reply:
I'll update it to check if it's really a bug.

Reply:
mmmh... I'm not a trustful source apparently... ;o

Reply:
not about this kata: voile messed it up (uneditable through the edit panel) by playing with the end points to change it's rank. See here: https://github.com/codewars/codewars.com/issues/1939#issuecomment-536228782

Reply:
With proper Markdown https://www.codewars.com/kata/56941f177c0a52aef50000a2

Reply:
Ugh, I didn't see your comment and didn't realize it had that issue. Now it's in beta.

Reply:
OK, should be fixed now. I couldn't re-approve normally because it had open issues.